### ISSUE TITLE
Zapojit se do mezinárodních PyLadies

### ISSUE THREAD

**Issue by encukou** [PyLadiesCZ/documents#19]
PyLadies mají svoje stránky, mailing list, materiály, atd.  Bylo by dobré, když už máme organizátorky, vést českou pobočku trochu oficiálněji.

**Comment by lspdv** [comment]
S tím naprosto souhlasím. Přes nadcházející volno, se o to mohu začít víc zajímat.

**Comment by lspdv** [comment]
Mohu dostat kontakt na organizátorky v Brně?

**Comment by encukou** [comment]
@Krejdom, budeš oficiální brněnská organizátorka? Já to být nemůžu, mám fousy :)

Ale stejně bych se rád podíval na úvodní mail, než ho pošlete, a trochu ho zrevidoval. A měly by o tom vědět i Zuzka z Terkou z Prahy :)

**Comment by lspdv** [comment]
To je určitě samozřejmost.

Zuzka a Tereza z Prahy to vidět samozřejmě mohou, pokud budou mít zájem to
vidět. Nutit nikoho nemůžu :-) Také neříkám, že to musím dělat já. Můhu
udělat nějakou rešerši a pak jim to předat ať se pod to podepíšou. Normálně
takto běžně funguji a nemám s tím problém.

Dne 16. prosince 2015 13:28 Petr Viktorin notifications@github.com
napsal(a):

**Comment by encukou** [comment]
Jasně :) Jenom říkám co je potřeba; komunikoval jsem s nimi už dřív. Myslím že jestli na to máš energii, vezmi si to na starosti, jen by ostatní měly mít možnost se vyjádřit.

V podstatě jde o to, představit se na pyladies@googlegroups.com, objevit se [tady](http://www.pyladies.com/locations/) (Brno už tam je), dostat organizátorskou adresu brno/praha@pyladies.com. Obecný návod je na http://kit.pyladies.com/en/latest/. (Ale něco je už přeskočené, protože PyLadies Brno začly dřív než Audrey a Lynn sepsaly ten návod).

**Comment by lspdv** [comment]
Ano, já s nimi diskutuji vše.

Dne 16. prosince 2015 14:23 Petr Viktorin notifications@github.com
napsal(a):

**Comment by honzajavorek** [comment]
Pokud to chápu správně, potřebujeme udělat dvě věci:
- Mít kontaktní osobu pro PyLadies v Brně a pro PyLadies v Praze.
- Zaregistrovat se do mezinárodního "PyLadies katalogu".

Jinak Zuzka má GitHub, @zuzejk, u Terky nevím. Můžeme věci diskutovat tady - od toho se to tu udělalo, mám pocit :)

**Comment by Krejdom** [comment]
Petře, bude mi ctí být organizátorkou za Brno. Vousy neplánuju, tak by to snad mohlo projít.

**Comment by encukou** [comment]
Super! :)

Ještě @ximara by se tady taky mohla chtít zapojit

**Comment by Ximara** [comment]
Jo, taky se ráda zapojím, vlastně jsem ti to v září slíbila. Bohužel jsem teď neměla moc času, ono to psaní bakalářky dalo zabrat…
Doufám, že se mi všechno podaří po státnicích dohnat. :)

**Comment by lspdv** [comment]
Svetlanka @SvetlanaM uz vicemene opravdu dokoncuje stranky pro PyLadies. Behem par dni bychom tu meli spolecne schvalit finalni podobu a obsah a proto bych jeste chvilku s tim predstavovanim se na googlegroups pockala, az budeme mit i stranky??? Mohlo by to vypadat celkove vic cool, kdyz to bude i se strankami, kde jsou obe meste a prostor pro dalsi mesta :-))) @encukou @Ximara @Krejdom @zuzejk

**Comment by encukou** [comment]
@lspdv, to zní rozumně. Předpokládám že ses o tom bavila i s Jacqueline.

Jinak, @pyterka už tu má účet, tak ji můžeme spamovat taky. (A maily by měly chodit všem, koho někdo označil nebo kdo přidal příspěvek, takže není potřeba označovat znova.)

**Comment by lspdv** [comment]
Skvele, tim oznacovanim/mailama jsem si nebyla jista. Diky!

S Jacqueline jsme se bavily o spouste veci i se Zuzcou, konkretne tohle mi spis rozmlouvala. Chtela nas pridat do skupiny rovnou a bez predstaveni, ze se predstavime az potom, ale mozna by bylo lepsi se radne predstavit a to i s novym webem, tak jsem to s dikem prozatim odmitla a obhajila. 

Se Svetlankou jsem web vcera prochazela, cimz jsem vas nechtela vynechat, ale Svetla chtela pohled nekoho, kdo by pripadne jako novacek hledal info na strankach, aby to pro nej nebyl chaos - coz ja porad jsem a snad jsme to trochu prilepsily. A domluvily jsme se, ze v utery si to ukazeme vsichni spolecne po PyLadies a doladime to, schvalime to:-) tak doufam, ze se k nam muzes pridat, Petre :-)

**Comment by encukou** [comment]
Jasně :)

**Comment by honzajavorek** [comment]
Ad vynechavani, ja bych se toho moc nebal. Ono je nekdy lepsi, kdyz se na
necem podili ze zacatku min lidi, rychleji se domluvi, vytvori nejakej
vysledek a nezdrzujou se velkym rozhodovanim a schuzovanim. Kdyz to je
hotovy v nejake prvni verzi, tak se to ukaze ostatnim a ti to muzou
poopravit nebo opripominkovat a vylepsit, ale to uz ve chvili, kdy neco je
a funguje. Takze za me v tomhle plus, klidne vynechavejte :)

**Comment by honzajavorek** [comment]
Jen upozorním, že toto tedy čeká hlavně na nový web. Jeho dokončování probíhá zde: https://github.com/PyLadiesCZ/pyladies.cz/issues Kdo chce, může přiložit Pull Request k dílu :)
